### PR TITLE
archimede: init at 0.0.2

### PR DESCRIPTION
**Pull request by anugrahn1**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by steeleduncan — CHANGES_REQUESTED** [review]

**Review by anugrahn1 — COMMENTED** [review]
I added the suggested changes, should I squash the commits for package.nix?

**Comment by steeleduncan** [comment]
Yes, you need to amend the commit and force push so that there are two commits (one adding maintainer, one adding the package), but it seems you already did that, so as far as I can tell, it is all in order!

**Comment by pluiedev** [comment]
I think you accidentally put the two commits in the wrong order... x)

File: pkgs/by-name/ar/archimede/package.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    repo = "archimede";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7P7PtzYlcNYG2+KW9zvcaRlTW+vHw8jeLD2dEQXmrzc=";
+  };
+
+  vendorHash = "sha256-F74TVp6+UdV31YVYYHWtdIzpbbiYM2I8csGobesFN2g=";
+
+  meta = {
+    homepage = "https://github.com/gennaro-tedesco/archimede";
+    description = "Unobtrusive directory information fetcher ";
```
Comment (FliegendeWurst) [inline pkgs/by-name/ar/archimede/package.nix:25]:
```suggestion
    description = "Unobtrusive directory information fetcher";
```

File: pkgs/by-name/ar/archimede/package.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    license = lib.licenses.asl20;
+    mainProgram = "archimede";
+    maintainers = [ lib.maintainers.anugrahn1 ];
+  };
+
```
Comment (FliegendeWurst) [inline pkgs/by-name/ar/archimede/package.nix:30]:
```suggestion
```

**Review by FliegendeWurst — CHANGES_REQUESTED** [review]

**Review by anugrahn1 — COMMENTED** [review]
I have made the changes, let me know if this looks good now.

**Review by pluiedev — APPROVED** [review]
Looks good, now we just have to wait for @ofborg

**Review by FliegendeWurst — APPROVED** [review]

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**